### ISSUE TITLE
Streamline Tseries (on behalf of the GUI)

### ISSUE THREAD

**Issue by rubencalje** [pastas/pasta#21]
To make PASTA more general, and to make it easier to generate the GUI and to be able to expand the GUI, a more general way to define Tseries would be nice (this is probably what Frans meant last week). For example, each Tseries should contain an attribute which states how many timeseries it needs (0 for Constant, 1 for Tseries1 and 2 for Recharge), and the series should be a list of this size, not seperate inputs like in the Recharge-class. With this information it is much easier to make an all-purpose import-dialog for Tseries, but it is also more logical for people who use a script.

**Comment by raoulcollenteur** [comment]
I agree. I initially implemented it this way I think, but changed it to the current method in [this commit](https://github.com/pastas/pasta/commit/bbe32d20d74c817192118f747ab7c233db61d96f). 

Can't remember anymore why I did this but it should be so difficult to changes this again. If we store all stresses in a list, an attribute is not really necessary as you can then also ask for it ` len(list)`.

**Comment by raoulcollenteur** [comment]
I think we should be able to close this (general) issue as many changes have been made to address it:

- Added a TseriesBase class from which all tseries inherit
- All tseries objects now have the same methods
- All tseries objects now have a get_stress method to get the stresses (very usefull for the GUI I presume).

Regarding the issue with different input data needed for all tseries objects, there might be room for some improvements. Although different tseries objects simply need different input data so I am not sure if this is really something we should work on now.

Cheers, Raoul
